src/ray/util/process.h

```diff
@@ -81,6 +81,10 @@ class Process {
   /// Convenience function to run the given command line and wait for it to finish.
   static std::error_code Call(const std::vector<std::string> &args,
                               const ProcessEnvironment &env = {});
+  /// Executes command line operation.
```

Comment:
```
  /// Executes command line operation.
  ///
  /// \param[in] argv The command line command to execute.
  /// \return The output from the command.
```

Reply:
Curious if we should somehow bake this into the formatter?  I see other parts of the code doesn't have this extra line and is more maintenance-free to have the formatter to this as opposed to human

Reply:
Also mention that it will throw exceptions in case of failures.